spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/FindHiddenMethodTest.java

```diff
@@ -0,0 +1,177 @@
+package edu.umd.cs.findbugs.detect;
+
+import static edu.umd.cs.findbugs.test.CountMatcher.containsExactly;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItem;
+
+import org.junit.jupiter.api.Test;
+import edu.umd.cs.findbugs.AbstractIntegrationTest;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
+
+class FindHiddenMethodTest extends AbstractIntegrationTest {
+    private final String BUG_TYPE = "HSM_HIDING_METHOD";
+
+    @Test
+    void testBadFindHiddenMethodTest() {
```

Comment:
For the "bad" testcases, there could be an assert for the number of the found buginstances as well.